src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -278,7 +279,6 @@ public void assignPersonToTeam(Person person, TeamName teamName) throws Duplicat
     public void removePersonFromTeam(Person person, TeamName teamName) throws PersonNotFoundException {
         if (!person.getTeamName().toString().equals(UNSPECIFIED_FIELD)) {
             try {
-                System.out.println(teams.getTeam(teamName).toString());
```

Comment:
Good that you managed to catch this "debug" print statement that should not be in production code.